templates/openshift/rbac/role.yaml

```diff
@@ -7,6 +8,18 @@ rules:
       - ""
     resources:
       - configmaps
-      - secrets
     verbs:
       - '*'
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: backupjob-secret-reader
+  namespace: default
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - secrets
```

Comment:
We can also limit this to the specific AWS credential secret. Just need to know what this secret is going to be called.